HTML and CSS/Product listing page with filters/product-listing.css

```diff
@@ -0,0 +1,108 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;600&family=Nunito:wght@300;400;500;600;700&display=swap');
+
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-family: 'Nunito', sans-serif;
+}
+
+body{
+    background-color: #f1f3f6;
+}
+
+.sort-byname {
```

Comment:
A hover property to this element changing the cursor to pointer and bg color a little , might make it look more interractive? Hope this helps, 
Onkar Deshpande